src/components/Toast/useToast.tsx

```diff
@@ -0,0 +1,18 @@
+import { useCallback } from "react";
+import { useToastContext } from "./ToastContext";
+import { ToastNoticeProps } from "./ToastNotice";
+
+export interface UseToastHook {
+  showToast: (props: ToastNoticeProps) => void;
+}
+export function useToast(): UseToastHook {
+  const { setNotice } = useToastContext();
```

Comment:
Question: Who eventually calls `setNotice(undefined)` or something like that to mean "the toast isn't shown anymore?".

It kinda looks like the caller is expected to pass in an `onClose`, but we don't have a way for them to call `hideToast()`, and it also seems like we should probably just do that for them anyway.

...like I think was expecting the `ToastNotice` `X` `onClick` to call `setNotice(undefined)` to tell Beam itself "don't show this toast anymore" and then call `onClose` as an (optional) notification to the page that we're not showing its toast anymore (which, fwiw I'm wondering if we can just drop, if `Toast` / `ToastNotice` fully owns making the toast go away).

Reply:
yes, that makes sense. I spoke to Brandon about this in office hours last week, so I dropped `ToastNotice` and combined with `Toast` and also dropped the `onClose` so now when you click the X it just calls `setNotice(undefined)`.  makes things clean and simple. thank you!